src/components/Molecules/HeaderEsuWithIcon/HeaderEsuWithIcon.js

```diff
@@ -0,0 +1,101 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import Icon from '../../Atoms/SocialIcons/Icon/Icon';
+import HeaderIcons from './assets/HeaderIcons';
+
+import {
+  IconWrapper,
+  EsuWrapper,
+  CloseButton,
+  HeaderESU
+} from './HeaderEsuWithIcon.style';
+
+/* HeaderEsuWithIcon component */
+const HeaderEsuWithIcon = ({
+  campaign,
+  title,
+  topCopy,
+  successCopy,
+  privacyCopy
+}) => {
+  const [isESUOpen, setIsESUOpen] = useState(false);
+  const [isSuccess, setIsSuccess] = useState(false);
+
+  // TODO: need to figure out how to handle validation here
+  /* eslint-disable-next-line no-unused-vars */
+  const [isErroring, setIsErroring] = useState('');
+
+  /* Allow our ESU modal stuff to happen */
+  const handleESUClick = e => {
+    e.preventDefault();
+    // Toggle our 'opened' state
+    setIsESUOpen(!isESUOpen);
+  };
+
+  /* Break out ESU render into own function */
+  const renderESU = () => {
+    return (
+      <HeaderESU
+        title={title}
+        topCopy={topCopy}
+        successCopy={successCopy}
+        privacyCopy={privacyCopy}
+        isSuccess={isSuccess}
+        subscribe={() => setIsSuccess(!isSuccess)}
+        errorMsg={isErroring}
```

Comment:
Can we expose those props instead of handling them internally ?
That way, we can do the request to our backend, and validate inside `comicrelief-contentful` repo.,
or may be do the validation here instead, just like https://github.com/comicrelief/comicrelief-contentful/blob/master/src/components/EmailSignUp/EmailSignUp.js#L4.

Reply:
Can do! I think this was from before I saw how you were handling the existing ESU within Contentful, will make it work in the same way 👍